tools/build-tools/src/buildVersion/buildVersion.ts

```diff
@@ -90,6 +92,41 @@ export function getSimpleVersion(file_version: string, arg_build_num: string, ar
     return fullVersion;
 }
 
+type TagPrefix = string | undefined | "client" | "server";
```

Comment:
I was trying to get better types for the prefixes, but this was a failed attempt.